src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/InntektsperiodeValgGammel.tsx

```diff
@@ -0,0 +1,243 @@
+import {
+    EInntektsperiodeProperty,
+    ESamordningsfradragtype,
+    IInntektsperiode,
+    samordningsfradagTilTekst,
+} from '../../../../../App/typer/vedtak';
+import React, { Dispatch, SetStateAction } from 'react';
+import styled from 'styled-components';
+import InputMedTusenSkille from '../../../../../Felles/Visningskomponenter/InputMedTusenskille';
+import { harTallverdi, tilTallverdi } from '../../../../../App/utils/utils';
+import { useBehandling } from '../../../../../App/context/BehandlingContext';
+import MånedÅrVelger from '../../../../../Felles/Input/MånedÅr/MånedÅrVelger';
+import LeggTilKnapp from '../../../../../Felles/Knapper/LeggTilKnapp';
+import { ListState } from '../../../../../App/hooks/felles/useListState';
+import { FormErrors } from '../../../../../App/hooks/felles/useFormState';
+import { InnvilgeVedtakForm } from './InnvilgeVedtak';
+import { VEDTAK_OG_BEREGNING } from '../../Felles/konstanter';
+import { useApp } from '../../../../../App/context/AppContext';
+import { FieldState } from '../../../../../App/hooks/felles/useFieldState';
+import { Tooltip } from '@navikt/ds-react';
+import { v4 as uuidv4 } from 'uuid';
+import { EnsligFamilieSelect } from '../../../../../Felles/Input/EnsligFamilieSelect';
+import { EnsligErrorMessage } from '../../../../../Felles/ErrorMessage/EnsligErrorMessage';
+import FjernKnapp from '../../../../../Felles/Knapper/FjernKnapp';
+import { TextLabel } from '../../../../../Felles/Visningskomponenter/Tekster';
+import { HorizontalScroll } from '../../Felles/HorizontalScroll';
+
+const Grid = styled.div<{ lesevisning?: boolean }>`
+    display: grid;
+    grid-template-columns: ${(props) =>
+        props.lesevisning ? 'repeat(4, max-content)' : ' repeat(6, max-content)'};
+    grid-gap: 0.5rem 1rem;
+
+    .ny-rad {
+        grid-column: 1;
+    }
+`;
+
+const StyledInput = styled(InputMedTusenSkille)`
+    text-align: left;
+`;
+
+export const tomInntektsperiodeRad = (årMånedFra?: string): IInntektsperiode => ({
+    årMånedFra: årMånedFra || '',
+    endretKey: uuidv4(),
+});
+
+const LeggTilRadKnapp = styled(LeggTilKnapp)`
+    margin-top: 0.5rem;
+`;
+
+interface Props {
+    className?: string;
+    inntektsperiodeListe: ListState<IInntektsperiode>;
+    samordningsfradragstype: FieldState;
+    samordningValideringsfeil?: FormErrors<InnvilgeVedtakForm>['samordningsfradragType'];
+    setValideringsFeil: Dispatch<SetStateAction<FormErrors<InnvilgeVedtakForm>>>;
+    skalVelgeSamordningstype: boolean;
+    valideringsfeil?: FormErrors<InnvilgeVedtakForm>['inntekter'];
+}
+
+const InntektsperiodeValgGammel: React.FC<Props> = ({
```

Comment:
```suggestion
const InntektsperiodeValgDeprecated: React.FC<Props> = ({
```
Men ja.. folk skjønner nok tegningen med nåværende navn også :)

Reply:
Dette navnet sa meg bare at denne komponenten skal fjernes på sikt, uten at jeg måtte se bruken av den først.

Reply:
https://github.com/navikt/familie-ef-sak-frontend/pull/2210/commits/c56877ffdc9cf290e50bff39c1278c1167ab637a